### PR TITLE
In 34 music

### PR DESCRIPTION
**Pull request by rox51124**

### CODE REVIEW

**Review by iDommel — COMMENTED** [review]
Nice job, please merge dev and fix conflicts before approval

File: src/Music.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        if (_music->frameCount == 0)
+            throw MusicError("Music file not found");
+    }
+
+    Music::~Music()
+    {
+        if (!_music)
+            return;
+        UnloadMusicStream(*_music);
+        delete _music.release();
```
Comment (iDommel) [inline src/Music.cpp:27]: I don't think you need to specifically delete this as it is a unique ptr

File: src/Music.hpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            ~Music();
+
+            enum class MusicState {
+                PLAY,
+                PAUSE,
+                STOP
+            };
+
+            ///  @brief Start music
+            void PlayMusic();
```
Comment (iDommel) [inline src/Music.hpp:31]: don't capitalize method names, that's C style coding

**Review by iDommel — CHANGES_REQUESTED** [review]
Some minor changes on method names

**Review by iDommel — APPROVED** [review]

**Review by MauxC — APPROVED** [review]

**Comment by iDommel** [comment]
I Just fixed the merge conflicts, I'm merging now

**Pull request merged**